.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+const config = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ['airbnb-base'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  plugins: ['@typescript-eslint'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/TheCodeCartel/mergify-test-service/pullRequest?prid=7838955)
```suggestion
  plugins: ["@typescript-eslint"],
```